src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+import UserService from '../services/userServices';
+import Util from '../utils/response';
+import Helper from '../middlewares/index';
+const { findUserById, updateUser } = UserService;
+const util = new Util();
+
+/** Class representing user controller. */
+class UserController {
+  /**
+  * get user profile
+  * @param {Object} req - server request
+  * @param {Object} res - server response
+  * @returns {Object} - custom response
+  * @description get details of registered user
+  */
+  static async getUserProfile(req, res) {
+    const { id } = req.params;
+    try {
+      const user = await findUserById(id);
+      const token = Helper.generateToken(id);
+      console.log(token)
+      if (!user) {
+        util.setError(401, `User with id: ${id} not found`);
+        return util.send(res);
+      }
+      util.setSuccess(200, 'Succesfully found user', user);
+      return util.send(res);
+    } catch (error) {
+      console.log('dssad')
```

Comment:
ditto

Reply:
Thanks for pointing this out 
It's a mistake

Reply:
Why the console.log?